client/my-sites/google-my-business/stats/index.js

```diff
@@ -66,36 +66,66 @@ class GoogleMyBusinessStats extends Component {
 		} );
 	};
 
-	renderViewsTooltipForDatanum = ( datanum, dataSeries ) => {
-		const { value: valueCount, date } = datanum;
-		if ( dataSeries.length > 20 ) {
-			return this.props.translate( '%(value)d view on %(day)s', '%(value)d views on %(day)s', {
-				count: valueCount,
-				args: {
-					value: valueCount,
-					day: moment( date ).format( 'D MMMM YYYY' ),
-				},
-				comment: 'How many views per day for a Google My Business location with date.',
-			} );
+	renderViewsTooltipForDatanum = ( datanum, interval ) => {
+		const { value: viewCount, date } = datanum;
+		if ( interval === 'quarter' ) {
+			return this.props.translate(
+				'%(viewCount)d view on the week of %(monday)s',
+				'%(viewCount)d views on the week of %(monday)s',
+				{
+					count: viewCount,
+					args: {
+						viewCount,
+						monday: moment( date ).format( 'LL' ),
+					},
+				}
+			);
+		} else if ( interval === 'week' ) {
+			return this.props.translate(
+				'%(viewCount)d view on %(day)s',
+				'%(viewCount)d views on %(day)s',
+				{
+					count: viewCount,
+					args: {
+						viewCount,
+						day: moment( date ).format( 'LL' ),
+					},
+				}
+			);
 		}
 
-		return valueCount;
+		return viewCount;
 	};
 
-	renderActionsTooltipForDatanum = ( datanum, dataSeries ) => {
-		const { value: valueCount, date } = datanum;
-		if ( dataSeries.length > 20 ) {
-			return this.props.translate( '%(value)d action on %(day)s', '%(value)d actions on %(day)s', {
-				count: valueCount,
-				args: {
-					value: valueCount,
-					day: moment( date ).format( 'D MMMM YYYY' ),
-				},
-				comment: 'How many actions per day for a Google My Business location with date.',
-			} );
+	renderActionsTooltipForDatanum = ( datanum, interval ) => {
+		const { value: actionCount, date } = datanum;
+		if ( interval === 'quarter' ) {
+			return this.props.translate(
+				'%(actionCount)d action on the week of %(monday)s',
+				'%(actionCount)d action on the week of %(monday)s',
+				{
+					count: actionCount,
+					args: {
+						actionCount,
+						monday: moment( date ).format( 'LL' ),
+					},
+				}
+			);
+		} else if ( interval === 'week' ) {
+			return this.props.translate(
+				'%(actionCount)d action on %(day)s',
```

Comment:
Hi! I've found a possible matching string that has already been translated **2** times:
`translate( '%(value)d action on %(day)s', '%(value)d actions on %(day)s' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).